pages/api/profile/experience/add.ts

```diff
@@ -9,12 +12,37 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
             return res.status(400).json({ status: 'error', message: 'body param missing' });
         }
 
-        const { companyName, position, startDate, endDate, summary, techStack } = JSON.parse(
-            req.body
-        );
+        const { userId, companyName, position, startDate, endDate, summary, techStack } =
+            JSON.parse(JSON.stringify(req.body));
 
-        if (!companyName || !position || !startDate) {
+        if (!userId || !companyName || !position || !startDate) {
             return res.status(400).json({ status: 'error', message: 'required param missing' });
```

Comment:
@JRakin please dont use status in  json.  status is reserved for statusCode of the api.